### PR TITLE
REF comments from Erin

### LINKED ISSUE #206: reuse runners + bootstrappers in joint fits
cc @esheldon

### PR DESCRIPTION
**Pull request by beckermr**
closes #206

### CODE REVIEW

File: metadetect/fitting.py [hunk 1]
```diff
@@ -613,6 +689,9 @@ def fit_mbobs_list_wavg(
 
         For Gaussians, this relationship is equivalent to smoothing by a round
         Gaussian with FWHM `fwhm_reg`.
+    symmetrize : bool, optional
+        If True, apply 4-fold symmetry to the mask+weight map. Default is True.
+        Ignored for gaussian fits.
```
Comment (beckermr) [inline metadetect/fitting.py:694]:
```
```

File: metadetect/fitting.py [hunk 2]
```diff
@@ -669,6 +750,9 @@ def fit_mbobs_wavg(
...
```
Comment (beckermr) [inline metadetect/fitting.py:755]:
```
```

**Review by beckermr — COMMENTED** [review]
Fix docs

File: metadetect/fitting.py (hunk 1 above)
Comment (beckermr) [inline metadetect/fitting.py:694]:
```suggestion
```

File: metadetect/fitting.py (hunk 2 above)
Comment (beckermr) [inline metadetect/fitting.py:755]:
```suggestion
```

**Review by beckermr — COMMENTED** [review]
Fix docs

**Review by esheldon — COMMENTED** [review]
Only style/coding comments

File: metadetect/fitting.py [hunk 3]
```diff
@@ -79,17 +90,35 @@ def fit_mbobs_gauss(
...
+                shear_mbobs.append(mbobs[band])
+
+    if flags == 0:
         try:
             ores = bootstrap(
                 shear_mbobs,
-                _make_obj_runner(rng, shear_mbobs),
-                psf_runner=_make_psf_runner(rng),
-                ignore_failed_psf=False,
+                get_gauss_obj_runner(
```
[the remark this answers is not in the document]
Comment (beckermr) [reply metadetect/fitting.py:111]: gonna pass on this one since having the function just figure it out is useful.

File: CHANGELOG.md [hunk 4]
```diff
@@ -10,6 +10,15 @@
 
 ### fixed
 
+## 0.11.0 - 2022-12-05
+
+### changed
+
+ - Gaussian fits are now done on coadds of the shear bands.
```
Comment (beckermr) [inline CHANGELOG.md:17]:
```suggestion
 - Gaussian fits are now done on coadds instead of a join fit across all shear bands.
```

**Pull request merged**